tools/diffscripts/diffapi.ps1

```diff
@@ -124,32 +124,32 @@ if ($null -eq $asmToolExecutableCommand)
 
 # Set the path to the SDK Internals Repo, if we specify the path on the command line we will set it as such
 $internalRootPath = ''
-if ($null -ne $SDKInternalsPath) 
+if ($null -ne $SDKInternalsPath)
 {
-    Write-Verbose "Using user supplied iot-sdk-internals repository." 
+    Write-Verbose "Using user supplied iot-sdk-internals repository."
     $internalRootPath = $SDKInternalsPath
-} 
-else 
+}
+else
 {
     $internalRootPath = Join-Path -Path $baseRootPath -Child "\iot-sdks-internals"
 }
 
 Write-Verbose "Using $internalRootPath for the internals sdk repository base directory."
 
 # If we specify to use the preview directory on the command line we will set it as such
-$compareDirectory = Join-Path -Path $internalRootPath -Child "\sdk_design_docs\CSharp\main"
-if ($IsPreview) 
+$compareDirectory = Join-Path -Path $internalRootPath -Child "\sdk_design_docs\CSharp\previews-v2"
```

Comment:
shouldn't this be main and the one in L143 be previews-v2?

Reply:
This will become main. It isn't a preview of our GA SDK v1.

I think we'll need to rename the current main directory to v1, and main will become v2.

Or something like that.

Thoughts?

Reply:
I see, that makes sense. We should probably also cut another preview at some point soon, else this previews/v2 -> preview diff will end up showing us a lot of outdated diffs (the preview branch hasn't been updated in a long long time). The only features in "preview" are dps client certificates and device streaming, both of which have their individual preview feature branches.

Reply:
We have nothing to preview besides v2, right?

Reply:
Nothing else right now, no.